mlrun/db/httpdb.py

```diff
@@ -950,12 +950,16 @@ def delete_runtime_resources(
         :param force: Force deletion - delete the runtime resource even if it's not in terminal state or if the grace
             period didn't pass.
         :param grace_period: Grace period given to the runtime resource before they are actually removed, counted from
-            the moment they moved to terminal state.
+            the moment they moved to terminal state (default is 4 hours).
```

Comment:
this is problematic to rely on the default value of the configuration, as the configuration can be easily changed on BE while running and this comment won't be relevant anymore. I you want to be accurate, then pass that param through the client-spec.